packages/types/src/index.ts

```diff
@@ -202,6 +202,8 @@ export type Alert = {
   alertDescriptionText?: string;
   alertUrl?: string;
   effectiveStartDate?: number;
+  /** Returned by OTP2 graphql queries, but not by OTP1 */
+  id?: string;
```

Comment:
I have been of the mind that we are removing OTP1 support entirely. Why don't we just make this field mandatory?